app/src/ui/changes/changes.tsx

```diff
@@ -41,7 +43,7 @@ export class Changes extends React.Component<IChangesProps, {}> {
             repository={this.props.repository}
             imageDiffType={this.props.imageDiffType}
             file={file}
-            readOnly={false}
+            readOnly={isCommitting ? true : false}
```

Comment:
```suggestion
            readOnly={isCommitting}
```
`this.props.isCommiting` is already a boolean.